Automattic/wp-calypso
# Hosting dashboard: Rework site logs header and toolbar

Fixes https://github.com/Automattic/wp-calypso/issues/96792

## Proposed Changes

This PR reworks the header and toolbar in the logs tab in the hosting dashboard per @fditrapani's design in #96792. Summary:

- Move the `Download logs` button to the header
- Use `ToggleGroupControl` from `@wordpress/components` for the log type switcher
- Move the labels above the controls (instead of to the left, as they previously were)
- Improve alignment of auto-refresh toggle
- General CSS cleanup

| New | Old |
| - | - |
| ![logs-php-new](https://github.com/user-attachments/assets/bd71960b-fa3f-4163-a216-e83a0af1e2cf) | ![logs-php-old](https://github.com/user-attachments/assets/88c66b0a-6c15-4146-854e-e233c191e796) |
| ![logs-web-new](https://github.com/user-attachments/assets/21b17bfb-914e-413b-a144-495bc657f113) | ![logs-web-old](https://github.com/user-attachments/assets/40bcd963-0b59-451d-9873-fa90a0c3f223) |


## Why are these changes being made?


See #96792

## Testing Instructions



1. Have a >=Business site
2. Navigate to `/site-logs/:site_slug/php`
3. Ensure that the page looks good and that the filter controls (including auto-refresh) works as expected
4. Ensure that the `Download logs` button works as expected
5. Switch to `Web server` logs
6. Ensure that the page looks good and that the filter controls works as expected
7. Shrink the viewport to mobile and ensure that things still work as expected
8. Navigate to `/sites/tools/logs/:site_slug/php`
9. Ensure that the page looks good in this context, too

## Pre-merge Checklist
